sgx-sys/src/lib.rs

```diff
@@ -106,15 +106,15 @@ mod test {
     #[cfg_attr(not(has_sgx), ignore)]
     #[test]
     fn enclave_create() {
-        use sgx_types::secs::*;
+        use sgx_types::*;
 
         let mut secs = Secs::default();
         secs.size = 8192;
         secs.base = 8192;
         secs.ssa_frame_size = 4096;
-        secs.attributes = Attributes::MODE_64_BIT;
-        secs.miscselect = MiscSelect::EXINFO;
-        secs.xfrm = Xfrm::X87 | Xfrm::SSE;
+        secs.attributes = attributes::Attributes::MODE_64_BIT;
+        secs.miscselect = miscselect::MiscSelect::EXINFO;
+        secs.xfrm = xfrm::Xfrm::X87 | xfrm::Xfrm::SSE;
```

Comment:
I don't think these lines are required anymore because these values are the defaults.

Reply:
It still wants `xfrm` specified to pass the test. I think this is because `SECS` defaults to `std::mem::zeroed()`.